content/en/platform/corda/1.2/cenm/deployment-kubernetes.md

```diff
@@ -275,18 +275,29 @@ For more details about joining CENM network see: [Joining an existing compatibil
 
 ### Display logs
 
-Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
+Each CENM service has a dedicated sidecar to displays live logs from the `log/` directory.
 
 To display logs use the following command:
 
 ```bash
-kubectl logs -c logs <pod-name>
+kubectl logs -c logs <logs-container> <pod-name>
+```
+
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
```

Comment:
Are you adding in: 
//For notary/pki/hsm is slightly different as it doesn't have a logs-notary container
kubectl logs <pod-name>   //for notary, pki, hsm